storage/rocksdb/ha_rocksdb.h

```diff
@@ -499,7 +499,9 @@ class ha_rocksdb : public my_core::handler {
   /* Array of index descriptors */
   std::shared_ptr<Rdb_key_def> *m_key_descr_arr;
 
-  bool check_keyread_allowed(uint inx, uint part, bool all_parts) const;
+  static bool check_keyread_allowed(bool &pk_can_be_decoded,
+                                    const TABLE_SHARE *table_share, uint inx,
```

Comment:
Is table_share arg required? It seems that it's always the same value - field table_share passed

Reply:
Yes, but the function is static after the changes. It's necessary to reuse the code outside the class.